lisa/operating_system.py

```diff
@@ -1922,7 +1922,9 @@ def __init__(self, node: Any) -> None:
         self._dnf_tool_name: str
 
     def _initialize_package_installation(self) -> None:
-        self.set_kill_user_processes()
+        self._log.debug(f"--- CBLMariner, type_name: {self._node.type_name()}")
+        if self._node.type_name() != "local":
```

Comment:
1. what does it need to skip local? if any reason, write it in the code comment.
2. use `isinstance to check types, not use a string`.